srcalc/src/main/java/gov/va/med/srcalc/web/controller/EnterVariablesController.java

```diff
@@ -44,12 +44,23 @@
     
     private final CalculationService fCalculationService;
     
+    /**
+     * Constructs an instance.
+     * @param calculationService the service to use for various operations when
+     *          performing a calculation
+     */
     @Inject
     public EnterVariablesController(final CalculationService calculationService)
     {
         fCalculationService = calculationService;
     }
     
+    /**
+     * Constructs a {@link VariableEntry} object with the variables needed for this calculation and 
```

Comment:
Good doc, but we should add that it also populates values from the most recent CalculationResut.